package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "d2l-test-reporting",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Will do first release manually since semantic release always wants to make the first version 1.0.0 and I'm not ready for that yet.